src/controllers.lisp

```diff
@@ -57,14 +56,17 @@
 
 (defmethod kit.sdl2:mousebutton-event ((instance sketch) state timestamp button x y)
   (let ((button (elt (list nil :left :middle :right) button))
-        (method (elt (list nil #'on-click #'on-middle-click #'on-right-click) button)))
+        (click-method (elt (list nil #'on-click-press #'on-middle-click-press #'on-right-click-press) button))
```

Comment:
hey! thanks for making the changes! I'm confused here now --- it's saying click methods are the press ones, and release methods are .. click?

why don't we do all three methods - press, click, release? I feel like release is interesting in a drag  & drop scenario

Reply:
Click methods are release, ya. That's how it works in JavaScript (`onclick()`) and I think it matches people's intuitions. E.g. if you press down the mouse button and move it around, you don't expect the click to take effect until you release the button. Could add an explicit `on-click-release` but it would be doing the same thing as `on-click`.